plugin.xml

```diff
@@ -43,6 +43,7 @@
         </config-file>
 
         <source-file src="src/android/Device.java" target-dir="src/org/apache/cordova/device" />
+        <framework src="com.github.ltrudu:DeviceIdentifiersWrapper:0.2" />
```

Comment:
Please move `framework` above `config-file` and add an empty line after it. The order should be `dependency`, `framework`, `podspec`, `config-file` then `source-file`/`header-file`, so dependencies come first and source files come last.